### ISSUE TITLE
[HW] Vikash Thakur

### ISSUE THREAD

**Issue by vikashth1** [datastaxdevs/appdev-week1-todolist#49]
**Name:** Vikash Thakur
**Email:** vikashth1@gmail.com
**Linkedin Profile:** https://www.linkedin.com/in/vikash-thakur-9b1992may/

Attach the homework screenshots below:
-----------------------------------------

<img width="1440" alt="Screenshot 2021-07-08 at 11 12 49 PM" src="https://user-images.githubusercontent.com/36493549/124972179-ad510500-e047-11eb-9ba9-672572383ae9.png">
<img width="1438" alt="Screenshot 2021-07-08 at 11 12 59 PM" src="https://user-images.githubusercontent.com/36493549/124972199-b4781300-e047-11eb-9eb5-68409867509d.png">
<img width="1440" alt="Screenshot 2021-07-08 at 11 32 57 PM" src="https://user-images.githubusercontent.com/36493549/124972277-c8237980-e047-11eb-8644-2a99bcc1ae7e.png">
<img width="1434" alt="Screenshot 2021-07-08 at 11 27 02 PM" src="https://user-images.githubusercontent.com/36493549/124972305-ceb1f100-e047-11eb-8b14-bef1068d8c8d.png">
<img width="1437" alt="Screenshot 2021-07-08 at 11 32 30 PM" src="https://user-images.githubusercontent.com/36493549/124972308-d1144b00-e047-11eb-96ef-77cb3be474ca.png">
<img width="1440" alt="Screenshot 2021-07-08 at 11 36 39 PM" src="https://user-images.githubusercontent.com/36493549/124972327-d6719580-e047-11eb-89f6-69d2b2435327.png">

I hope that's all for HW do let me know if is there anything else need to complete.

**Comment by SonicDMG** [comment]
Yes, you did everything we asked for, awesome job, thank you. Here's your badge -> https://api.badgr.io/public/assertions/fS4SXuGMTl26dt510YovXA?identity__email=vikashth1%40gmail.com

**Comment by HadesArchitect** [comment]
Hey **Vikash Thakur**, great job! Congrats! Here is your badge! https://api.badgr.io/public/assertions/GSnhX5lKQE6Fl7hRa5C-Xg
